src/app/components/info-message/info-message.component.spec.ts

```diff
@@ -1,20 +1,21 @@
-import { Component, Input, ViewChild } from '@angular/core';
+import { Component, CUSTOM_ELEMENTS_SCHEMA, Input, ViewChild } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
-import {InfoMessage, InfoMessageType} from '../../enums';
-import { WorkAllocationComponentsModule } from '../work-allocation.components.module';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```

Comment:
Path should be relative '../'. Should not start with 'src'. Also noticed in few other places below